### ISSUE TITLE
Qucsconv fails to convert SPICE netlist

### ISSUE THREAD

**Issue by ra3xdh** [ra3xdh/qucsator_rf#26]
See https://github.com/ra3xdh/qucs_s/issues/967 for context and test case. May appear after #25 or earlier. The Qucsconv v1.0.0 converts netlits without error. Need further investigations.

**Comment by ra3xdh** [comment]
The `git bisect` shows that this was broken after #25

**Comment by tomhajjar** [comment]
Another issue with qucsconv was it complaining about "no .END directive found" for every OpAmp Spice file in this project. 

It never caused issues but I couldn't figure out what the problem was.

![2024-09-28_093200](https://github.com/user-attachments/assets/f1deeb29-2cf8-4cdc-b974-3da65b1d0200)

[Opamp_Spice_qucs_prj.zip](https://github.com/user-attachments/files/17174535/Opamp_Spice_qucs_prj.zip)

**Comment by ra3xdh** [comment]
> Another issue with qucsconv was it complaining about "no .END directive found"

This message was removed in v1.0.0 by this commit: https://github.com/ra3xdh/qucsator_rf/commit/bfb98b18317a2415f3139d0cca97b593795830c9

**Comment by ra3xdh** [comment]
The spice to qucs library conversion also doesn't work after #25 Qucsconv became totally broken. I have to revert #25
